### PR TITLE
[s] Safer input for the station charter

### PR DESCRIPTION
**Pull request by optimumtact**
Prevents html entities and sets a maximum length of 30

### CODE REVIEW

**Comment by PKPenguin321** [comment]
so wait could you like put links into the station name and shit?

**Comment by KorPhaeron** [comment]
Hold on I still nees to link my patreon before you fix this

File: code/game/objects/items/charter.dm [hunk 1]
```diff
@@ -14,7 +14,7 @@
 		user << "The crew has already settled into the shift. It probably wouldn't be good to rename the station right now."
 		return
 
-	var/new_name = input(user, "What do you want to name [station_name()]? Keep in mind particularly terrible names may attract the attention of your employers.")  as text|null
+	var/new_name = stripped_input(user, "What do you want to name [station_name()]? Keep in mind particularly terrible names may attract the attention of your employers.")  as text|null
```
Comment (Jordie0608) [inline code/game/objects/items/charter.dm]: Maybe `reject_bad_name()` or `reject_bad_text()`?

**Comment by Bawhoppen** [comment]
can you also make it so it always say "SYBIL:" or "BASIL:" in front of it?

File: code/game/objects/items/charter.dm (hunk 1 above)
Comment (optimumtact) [inline code/game/objects/items/charter.dm]: why

Comment (Jordie0608) [inline code/game/objects/items/charter.dm]: They sanitize things to be more name-like than html_encode does.

Comment (optimumtact) [inline code/game/objects/items/charter.dm]: I'll take a look

**Comment by SidonCC** [comment]
Also a letter limit would be nice

File: code/game/objects/items/charter.dm (hunk 1 above)
Comment (optimumtact) [inline code/game/objects/items/charter.dm]: They didn't seem to do much more processing that I found useful, so someone else can add it if they want.

**Pull request merged**